### ISSUE TITLE
[IDE] Allow ability to navigate to a different view to see something and then come back to a generated fix

### ISSUE THREAD

**Issue by jwmatthews** [konveyor/kai#152]
At present I am not able to navigate away from a Generated Fix without losing it and needing to regenerate.

Steps:

1. Click Kai Fix All
2. See the Generated Fix
3. On the Left Tree Explorer, click another file to see contents
4. Notice the Generated Fix view is gone
5. How do we get the Generated Fix view back without regenerating?

Video to show the flow:

- https://www.loom.com/share/48bce677ce4d4df989246b5d2a277bc2?sid=50728222-c763-4163-aeb5-c5f1efd15a53

**Comment by shawn-hurley** [comment]
@djzager do you believe that this is completed. I think it is but want to ask.

**Comment by djzager** [comment]
Yes. This is addressed in https://github.com/konveyor/editor-extensions you can switch back-and-forth between the editor, webview, konveyor resolutions, and suggested fix.
